supplementary_style_guide/glossary_terms_conventions/general_conventions/w.adoc

```diff
@@ -28,6 +28,16 @@
 
 *See also*:
 
+[[weak-dependencies]]
+==== image:images/yes.png[yes] Weak dependencies (noun)
```

Comment:
make singular?

Reply:
@jseseCCS  I think this might be referred to in plural form
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/packaging_and_distributing_software/new-features-in-rhel-8_packaging-and-distributing-software